ssl/ssl_lib.c

```diff
@@ -2809,6 +2809,19 @@ int SSL_export_keying_material(SSL *s, unsigned char *out, size_t olen,
                                                        contextlen, use_context);
 }
 
+int SSL_export_keying_material_early(SSL *s, unsigned char *out, size_t olen,
+                                     const char *label, size_t llen,
+                                     const unsigned char *context,
+                                     size_t contextlen, int use_context)
+{
+    if (s->version != TLS1_3_VERSION) {
+        return -1;
```

Comment:
I think perhaps we should just return 0 for failure and 1 for success. I know `SSL_export_keying_material` can return 0 or -1 for failure - but I'm not sure the distinction between the two error codes is helpful.

Reply:
Done